pygeoapi/linked_data.py

```diff
@@ -173,104 +175,146 @@ def jsonldify_collection(cls, collection, locale_):
 
 def geojson2geojsonld(config, data, dataset, identifier=None, id_field='id'):
     """
-    Render GeoJSON-LD from a GeoJSON base. Inserts a @context that can be
-    read from, and extended by, the pygeoapi configuration for a particular
-    dataset.
+        Render GeoJSON-LD from a GeoJSON base. Inserts a @context that can be
+        read from, and extended by, the pygeoapi configuration for a particular
+        dataset.
 
-    :param config: dict of configuration
-    :param data: dict of data:
-    :param dataset: dataset identifier
-    :param identifier: item identifier (optional)
-    :param id_field: item identifier_field (optional)
+        :param config: dict of configuration
+        :param data: dict of data:
+        :param dataset: dataset identifier
+        :param identifier: item identifier (optional)
+        :param id_field: item identifier_field (optional)
 
-    :returns: string of rendered JSON (GeoJSON-LD)
+        :returns: string of rendered JSON (GeoJSON-LD)
     """
-    context = config['resources'][dataset].get('context', [])
-    geojsonld = config['resources'][dataset].get('geojsonld', True)
+    context = config['resources'][dataset].get('context', []).copy()
+    defaultVocabulary = {
+        'schema': 'https://schema.org/',
+        id_field: '@id',
+        'type': '@type'
+    }
 
     if identifier:
         # Single geojsonld
-        if not geojsonld:
-            data, geocontext = make_jsonld(data)
-            data[id_field] = identifier
-        else:
-            data['id'] = identifier
+        defaultVocabulary.update({
+            'geosparql': 'http://www.opengis.net/ont/geosparql#'
+        })
+
+        # Expand properties block
+        data.update(data.pop('properties'))
+
+        # Include multiple geometry encodings
+        data['type'] = 'schema:Place'
+        jsonldify_geometry(data)
+        data[id_field] = identifier
 
     else:
         # Collection of geojsonld
+        defaultVocabulary.update({
+                'features': 'schema:itemListElement',
```

Comment:
Odd indenting - linting?

Reply:
don't know how flake8 missed this. fixed now